packages/mobile/test/utils.ts

```diff
@@ -83,6 +83,17 @@ export function createMockNavigationProp(params: any): NavigationScreenProp<any>
   }
 }
 
+export function createMockNavigationProp2(params: any): NavigationScreenProp<any> {
```

Comment:
Can you just replace the old one instead of adding this second? Or at least make the differentiating more clear

Reply:
I wanted to replace it but it's a bit of a can of worms because `createMockNavigationProp` looks wrong to me and it is used very widely. Maybe I am missing something...can you take a look?

The only difference in `createMockNavigationProp2` is that I am passing the `property` through the `getParam` method so that I can grab the specific property rather than whole object of parameters.

Reply:
Took a closer look and I am pretty sure `createMockNavigationProp` doesn't work how it's intended. Fixing it would require fixing ~12 different test files, majority of which are not related to Secure Send. Do you think it's worth taking on in this PR or opening a separate issue for?

Reply:
Fair enough. How about you rename this to `createMockNavigationProp` and the other to `createMockNavigationPropDeprecated` and leave a comment to clarify?

Reply:
Works for me!